.github/workflows/release-editor-kit.yml

```
@@ -0,0 +1,61 @@
+      - name: Release editor-preload package and Sync to CDN
+        uses: galacean/publish@main
+        with:
+          publish: false
+          packages: |
+            editor-preload
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+          NPM_CONFIG_PROVENANCE: true
+          OASISBE_UPLOAD_URL: https://oasisbe.alipay.com/api/file/no-auth/crypto/upload
+          OASISBE_REQUEST_HEADER: ${{secrets.OASISBE_REQUEST_HEADER}}
+          OASISBE_PUBLIC_KEY: ${{secrets.OASISBE_PUBLIC_KEY}}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)